app-backend/batch/src/main/scala/com/azavea/rf/batch/export/CheckExportStatus.scala

```diff
@@ -54,24 +54,26 @@ case class CheckExportStatus(exportId: UUID, statusBucket: String = "rasterfound
         }
       }
 
+    // This changed to OptionT because of strange compilation behavior after bumping the cats version
+    // I'd like to investigate further, but it looks like it might be a bug
     val result = for {
```

Comment:
Some of these likely go away once we replace `slick` too -- at least I expect some of this code will need to change

Reply:
That's probably true. I'm also going to write up an issue for Cats at some point next week if I can produce a minimal example of the issue here